app/Domain/Goalcanvas/Templates/canvasComment.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends($layout)
+@section('content')
+
+@php
+
+/**
+* canvasComment.inc template - Generic template for comments
+*
+* Required variables:
+* - $canvasName Name of current canvas
+*/
+
+
+$canvasName = "goal";
```

Comment:
Please replace the canvasName variable with "Goal" throughout the module. This should not be in there anymore